packages/dnb-eufemia/src/components/upload/Upload.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Components
+import Lead from '../../elements/Lead'
+
+// Shared
+import { createSpacingClasses } from '../space/SpacingHelper'
+import { createSkeletonClass } from '../skeleton/SkeletonHelper'
+import Context from '../../shared/Context'
+import { extendPropsWithContext } from '../../shared/component-helper'
+import { format } from '../number-format/NumberUtils'
+
+// Internal
+import UploadFileInput from './UploadFileInput'
+import type { UploadFile, UploadProps } from './types'
+import UploadFileListCell from './UploadFileListCell'
+import useUpload from './useUpload'
+
+export const defaultProps = {
+  fileMaxSize: 5000,
+}
+
+const Upload = (localProps: UploadProps) => {
+  const context = React.useContext(Context)
+
+  const extendedProps = extendPropsWithContext(
+    localProps,
+    defaultProps,
+    { skeleton: context?.skeleton },
```

Comment:
```suggestion
    { skeleton: context?.skeleton },
    context.getTranslation(props).Upload,
```

I still recommend we follow this patterns, because this way a dev and send in the `locale` or `lang` prop, and define/change (overwrite) the locale for only this component. We will add these props to the docs and the prop types. There is a task for that 👍